README.ru.md

```diff
@@ -0,0 +1,353 @@
+<!-- markdownlint-disable MD001 MD012 MD026 -->
```

Comment:
Any reason we have to disable these rules?

Reply:
Just because the English README doesn't enforce markdownlint rules and has quite a few green squiggly lines.

In `README.ru.md` I fixed what I could without changing the structure too much, but some other I left. 

MD0001 is disabled because FAQ section is H2 and the next section is H4 skipping H3, for example.

Perhaps there should be a markdownlint should be included in CI for all PRs in this repo.

Reply:
Yup but I won't force you to do that.